bundles/org.openhab.ui/web/src/pages/settings/model/model.vue

```diff
@@ -376,6 +385,20 @@ export default {
         this.ready = true
         this.$nextTick(() => { this.initSearchbar = true })
         if (!this.eventSource) this.startEventSource()
+
+        if (this.empty) {
+          let dialog = this.$f7.dialog.confirm('There is no semantic model. Create model locations from template?', () => {
+            this.$f7router.navigate({
+              url: 'add-template',
+              route: {
+                component: AddFromTemplate,
+                path: 'add-template',
+                props: {
+                }
+              }
+            })
+          })
+        }
```

Comment:
I would rather not have this, to be honest.
The button in the empty state placeholder is enough, IMO.

Reply:
> I would rather not have this, to be honest.

Done. PR should be up to date now as well.